tracing/providers/jaeger/src/main/java/io/helidon/tracing/providers/jaeger/JaegerTracerProvider.java

```diff
@@ -15,11 +15,15 @@
  */
 package io.helidon.tracing.providers.jaeger;
 
+import java.util.List;
 import java.util.Optional;
+import java.util.ServiceLoader;
 
+import io.helidon.common.HelidonServiceLoader;
 import io.helidon.common.Weight;
 import io.helidon.common.Weighted;
 import io.helidon.tracing.Span;
+import io.helidon.tracing.SpanLifeCycleListener;
```

Comment:
General nit: "lifecycle" is one word, so `Lifecycle`, not `LifeCycle`

Reply:
Additionally, consider the Java Beans-like listener conventions and lose the whole notion of lifecycle. Then you have `SpanListener` whose methods are (ideally, in current Helidonic convention, `started` etc.). If the "before" event is the problem, then perhaps dwell on that in isolation.

Reply:
Yup. Good point. Will change.